--- source/src/C++/AtomicCount.h	1969-12-31 19:00:01.000000000 -0500
+++ source.2/src/C++/AtomicCount.h	2025-10-20 00:26:16.218416180 -0400
@@ -28,16 +28,17 @@
 #include "Mutex.h"
 #endif
 
+#ifdef ENABLE_BOOST_ATOMIC_COUNT
+#include <boost/smart_ptr/detail/atomic_count.hpp>
+#endif
+
 namespace FIX
 {
   /// Atomic count class - consider using interlocked functions
 
 #ifdef ENABLE_BOOST_ATOMIC_COUNT
-
-#include <boost/smart_ptr/detail/atomic_count.hpp>
 typedef boost::detail::atomic_count atomic_count;
-
-#elif _MSC_VER 
+#elif _MSC_VER
 
   //atomic counter based on interlocked functions for Win32
   class atomic_count
